src/app/app.css

```
@@ -9,6 +10,23 @@
+     rem 으로 적는다. px 로 두면 Tailwind 가 rem 인 다른 구간보다 앞에 뽑아
+     내려 lg: 가 xl: 을 덮는다. */
+  --breakpoint-xl: 90rem;
+
+  /* 소개 랜딩 히어로 헤드라인만 쓰는 구간이다. 벡터라 글자를 접을 수 없어
+     390 을 기준으로 두 줄짜리와 세 줄짜리 파일이 갈린다. 390px = 24.375rem. */
+  --breakpoint-xs: 24.375rem;
+
+  /* 소개 랜딩 본문 전용. 시안이 이 문단들만 SUIT 로 잡았다. */
+  --font-suit:
+    "SUIT Variable", "Pretendard Variable", "Pretendard", -apple-system,
+    BlinkMacSystemFont, system-ui, Roboto, sans-serif;
```

Stylelint (17.14.0)

[error] Expected "BlinkMacSystemFont" to be "blinkmacsystemfont" (value-keyword-case)

(value-keyword-case)